src/data_storage/sqlalchemy/models.py

```diff
@@ -30,21 +31,24 @@ def get_connection_engine():
 class Story(Base):
     __tablename__ = "story"
 
-    id = Column(Integer, primary_key=True)
+    id = Column(String(36), primary_key=True)
     name = Column(String(200), nullable=False)
 
 
 class Node(Base):
+    """Make boolean flag to show root node or name always root node as Root"""
+
     __tablename__ = "node"
 
     id = Column(Integer, primary_key=True)
     story = Column(
-        Integer,
+        String(36),
         ForeignKey("story.id", ondelete="CASCADE"),
         nullable=True,
     )
     name = Column(String(200), nullable=False, index=True)
     text = Column(String(200), nullable=False)
+    is_root = Column(Boolean, server_default=expression.false(), nullable=False)
```

Comment:
Mb we should define in `Story` table one fields `root_node_id`, instead of column here?

Reply:
At first, I thought that, too. But getting CircularDependencyError(https://docs.sqlalchemy.org/en/14/core/exceptions.html?highlight=error#sqlalchemy.exc.CircularDependencyError). And this approach makes updating and deleting `Node` item complex.

Reply:
We can
1) Add `Story.root = Column(Integer)` without FK constraint. 
2) Or create `StoryRoot(story_id, root_id)` table.